javascript/javascript1/week4/tictactoeApp.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+         const position = [
+ ['o', ' ', ' '],
+ ['o', ' ', ' '],
+ ['o', ' ', ' ']
+ ];
+
+const position = [
+  [' ', ' ', 'o'],
+  [' ', ' ', 'o'],
+  [' ', ' ', 'o']
+];
+
+const position = [
+ ['o', 'o', 'o'],
+[' ', ' ', ' '],
+[' ', ' ', ' ']
+];
+
+const position = [
+[' ', ' ', ' '],
+ ['o', 'o', 'o'],
+[' ', ' ', ' ']
+ ]; 
+
+const position = [
+  [' ', ' ', ' '],
+  [' ', ' ', ' '],
+ ['o', 'o', 'o']
+];
+
+const position = [
+ ['o', ' ', ' '],
+ [' ', 'o', ' '],
+[' ', ' ', 'o']
+];
+
+const position = [
+ [' ', ' ', 'o'],
+[' ', 'o', ' '],
+['o', ' ', ' ']
+];
+
+const position = [
+  [' ', 'x', ' '],
+  [' ', 'x', ' '],
+  [' ', 'x', ' ']
+];
+
+console.log(position[0][1])
+console.log(position[1][1])
+console.log(position[2][1])
+
+function hasWon(position, player){
+	if( position[0][1] === player && position[1][1] === player  && position[2][1] === player ){
+		return true
+	}else if( position[0][2] === player && position[1][1] === player  && position[2][0] === player ){
+		return true
+	}
+
+	return false
+}
```

Comment:
Remember comparisons always evaluate to a boolean, so if you wanted, you can refactor this to be 
```suggestion
function hasWon(position, player){
    return (position[0][1] === player && position[1][1] === player  && position[2][1] === player)  || (position[0][2] === player && position[1][1] === player  && position[2][0] === player)
}
```